cmd/catalyst-node/catalyst-node_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+const (
+	closestNodeAddr = "someurl.com"
+	playbackID      = "abc123"
```

Comment:
Just to handle a few valid cases, perhaps
```suggestion
	playbackID      = "abc_XYZ-123"
```